### DISCUSSION TITLE
Turn off color in the validate CLI

### DISCUSSION THREAD

**Discussion by tbenade** [vectordotdev/vector#20078] [Q&A] [answered]
Hi, is there any way to turn of colored output when calling `vector validate` please.

Thanks

**Comment by jszwedko** [comment, marked as answer]
Hi @tbenade ,

You can use `--color=never` like `vector --color=never validate <file>`.

**Reply by tbenade** [reply]
aaah I came undone as I had it ` vector validate --config-dir ".build/development/app/vector/" --color=never` which rendered 
```
error: unexpected argument '--color' found

  tip: to pass '--color' as a value, use '-- --color'

Usage: vector validate <--no-environment|--deny-warnings|--config-toml <config-toml>|--config-json <config-json>|--config-yaml <config-yaml>|PATHS|--config-dir <config-dir>>
```

**Reply by jszwedko** [reply]
Ah, yeah, that's an idiosyncrasy of the CLI flag parsing. We hope to fix that as part of https://github.com/vectordotdev/vector/issues/12242
